driver-sync/src/test/functional/com/mongodb/client/unified/CommandMonitoringTest.java

```diff
@@ -38,6 +38,9 @@ public CommandMonitoringTest(@SuppressWarnings("unused") final String fileDescri
                                  final BsonDocument definition) {
         super(schemaVersion, runOnRequirements, entities, initialData, definition);
         assumeFalse(isServerlessTest());
+        // The driver has a hack where getLastError command is executed as part of the handshake in order to get a connectionId
```

Comment:
This code duplication between reactive and sync tests is unfortunate, but don't want to deal with that in this PR.